api/ccapi/include/model.h

```diff
@@ -40,6 +41,21 @@ enum class ModelType {
   UNKNOWN     /** Unknown */
 };
 
+/**
+ * @brief Model saving options
+ *
+ */
+enum class ModelSaveLoadFormat {
+  MODEL_SAVE_LOAD_FORMAT_BIN =
```

Comment:
```MODEL_FORMAT_BIN```?
(SAVE-LOAD is unnecessary duplicated terms)

Reply:
Yes, prefix save_load is deleted, across the patch.